docs/changehistory/NextVersion.md

```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| "properties": "\_none\_"      | ![Properties of Spatial Category are merged](./media/SpatialCategoryPropertiesMerged.png)                                         | ![Properties of Spatial Category not merged](./media/SpatialCategoryPropertiesNotMerged.png)                                      |
+| "properties": \["UserLabel"\] | ![Properties of Physical Object and Spatial Category not merged](./media/PhysicalObjectAndSpatialCategoryPropertiesNotMerged.png) | ![Properties of Physical Object and Spatial Category not merged](./media/PhysicalObjectAndSpatialCategoryPropertiesNotMerged.png) |
+
+### Fixed a bug that prevented disabling `relatedProperties` specifications
+
+The bug made it impossible to remove related properties if a lower priority rule specified that the property should be included.
+
+For example, it is now possible to remove all properties of a class `ExternalSourceAspect`, which is owned by a class `Element`. In order to do that, the following rules can be used:
+
+```json
+{
+  "ruleType": "ContentModifier",
+  "class": {
+    "schemaName": "BisCore",
+    "className": "Element"
+  },
+  "relatedProperties": [
+    {
+      "propertiesSource": {
+        "relationship": {
+          "schemaName": "BisCore",
+          "className": "ElementOwnsMultiAspects"
+        },
+        "direction": "Forward",
+        "targetClass": {
+          "schemaName": "BisCore",
+          "className": "ExternalSourceAspect"
+        }
+      },
+      "properties": []
+    }
+  ]
+}
+```
+
+```json
+{
+  "ruleType": "ContentModifier",
+  "class": {
+    "schemaName": "BisCore",
+    "className": "Element"
+  },
+  "relatedProperties": [
+    {
+      "propertiesSource": {
+        "relationship": {
+          "schemaName": "BisCore",
+          "className": "ElementOwnsMultiAspects"
+        },
+        "direction": "Forward",
+        "targetClass": {
+          "schemaName": "BisCore",
+          "className": "ExternalSourceAspect"
+        }
+      },
+      "properties": "_none_"
+    }
+  ]
+}
+```
+
+```json
+{
+  "ruleType": "ContentModifier",
+  "class": {
+    "schemaName": "BisCore",
+    "className": "Element"
+  },
+  "relatedProperties": [
+    {
+      "propertiesSource": {
+        "relationship": {
+          "schemaName": "BisCore",
+          "className": "ElementOwnsMultiAspects"
+        },
+        "direction": "Forward",
+        "targetClass": {
+          "schemaName": "BisCore",
+          "className": "ExternalSourceAspect"
+        }
+      },
+      "properties": [
+        {
+          "name": "Identifier",
+          "isDisplayed": false
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
I'd reword to this:

For example, the [default BisCore supplemental ruleset](https://github.com/iTwin/itwinjs-core/blob/504a7b88e9ade29cc306bd55d093be1d76ddad43/presentation/backend/assets/supplemental-presentation-rules/BisCore.PresentationRuleSet.json#L372-L419) automatically includes `ExternalSourceAspect.Identifier` property to all Elements' content when they have such aspect. It's now possible to disable that rule using one of these approaches:

- Set `properties` to `"_none_"` or `[]`:
  
  ```json
  {
    "ruleType": "ContentModifier",
    "class": { "schemaName": "BisCore", "className": "Element" },
    "relatedProperties": [{
      "propertiesSource": {
        "relationship": { "schemaName": "BisCore", "className": "ElementOwnsMultiAspects" },
        "direction": "Forward",
        "targetClass": { "schemaName": "BisCore", "className": "ExternalSourceAspect" }
      },
      "properties": "_none_"
    }]
  }
  ```

- Disable property display:

  ```json
  {
    "ruleType": "ContentModifier",
    "class": { "schemaName": "BisCore", "className": "Element" },
    "relatedProperties": [{
      "propertiesSource": {
        "relationship": { "schemaName": "BisCore", "className": "ElementOwnsMultiAspects" },
        "direction": "Forward",
        "targetClass": { "schemaName": "BisCore", "className": "ExternalSourceAspect" }
      },
      "properties": [{
        "name": "Identifier",
        "isDisplayed": false
      }]
    }]
  }
  ```